### PR TITLE
Cleanups prior to 1.1.0 release

### PR DESCRIPTION
**Pull request by MLnick**
* Bump to latest version of `max-base`
* Add test case for PNG image format
* Update model metadata in line with MAX standard and maxfw requirements

### CODE REVIEW

File: tests/test.py [hunk 1]
```diff
@@ -23,19 +23,15 @@ def test_metadata():
     assert r.status_code == 200
 
     metadata = r.json()
-    assert metadata['id'] == 'deeplab-tf'
-    assert metadata['name'] == 'deeplab TensorFlow Model'
-    assert metadata['description'] == 'deeplab TensorFlow model trained on VOCO 2012'
+    assert metadata['id'] == 'max-image-segmenter'
+    assert metadata['name'] == 'MAX Image Segmenter'
+    assert metadata['description'] == 'DeepLab TensorFlow model for semantic image segmentation, trained on VOCO 2012'
     assert metadata['license'] == 'Apache v2'
```
Comment (ptitzler) [inline tests/test.py]: Just checking: Is this `Apache v2`  our official standard value for the https://www.apache.org/licenses/LICENSE-2.0?

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM codait/max-base:v1.0.0
+FROM codait/max-base:v1.1.0
```
Comment (bdwyer2) [inline Dockerfile]:
```suggestion
FROM codait/max-base:v1.1.1
```

File: tests/test.py (hunk 1 above)
Comment (bdwyer2) [reply tests/test.py]: We have not been consistent about this. We should standardize on something.

Comment (ptitzler) [reply tests/test.py]: https://github.ibm.com/CODAIT/Production-Readiness/issues/21

**Pull request merged**